portal-api/src/v1/errors/invalid-report-status.error.ts

```diff
@@ -0,0 +1,6 @@
+export class InvalidReportStatusError extends Error {
```

Comment:
In the interest of this PR not taking on too many refactors I'm tempted to push against this for now

Reply:
That's fair. I left a comment about this elsewhere, but I think we should still include the specific http error code in this class as if it were using the `ApiError` class so that the controller doesn't need to hard code the `500` itself